.github/workflows/book.yml

```diff
@@ -1,8 +1,9 @@
-name: Build book
+name: Build Book
 
 on:
   push:
     branches: ['main']
+  workflow_dispatch:
```

Comment:
What is this for?

Reply:
Its to enable manual triggering. I needed it for testing and I think it can be useful.

Reply:
Okay!